spec/ErrorCode.schema.json

```diff
@@ -2,5 +2,5 @@
     "id": "ErrorCode",
     "type": "string",
     "minLength": 1,
-    "pattern": "^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*(:[a-zA-Z0-9_]+(-[a-zA-Z0-9_]+)*)*$"
+    "pattern": "^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*(:[a-zA-Z0-9_,\\s]+(-[a-zA-Z0-9_]+)*)*$"
```

Comment:
in the case of duplicate burden estimate rows, the error code contains a comma separated list of fields from the violated unique constraint